lib/cdo/delete_accounts_helper.rb

```diff
@@ -118,10 +118,18 @@ def clean_and_destroy_pd_content(user_id, user_email)
     Pd::Attendance.with_deleted.where(teacher_id: user_id).update_all(teacher_id: nil, deleted_at: Time.now)
     Pd::Attendance.with_deleted.where(marked_by_user_id: user_id).update_all(marked_by_user_id: nil)
 
-    Pd::Teachercon1819Registration.where(user_id: user_id).update_all(form_data: '{}', user_id: nil)
```

Comment:
Unlike many other records we anonymize in this file, we mark `user_id: nil` here. I couldn't find documentation on why we do that –– there was a `belongs_to :user, optional: true` in this (now removed) model, which follows the same pattern as other models like `Pd::Application::ApplicationBase` that anonymize records.

Reply:
good observation! I don't see any reason we would need to wipe the user id here. so, preserving the user_id is one possible fix

Reply:
Excellent –– doing that!